artemis-hawtio/artemis-plugin/src/main/webapp/plugin/js/artemisPlugin.js

```diff
@@ -96,6 +96,9 @@ var ARTEMIS = (function(ARTEMIS) {
          .when('/artemis/deleteAddress', {
            templateUrl: ARTEMIS.templatePath + 'deleteAddress.html'
          })
+         .when('/artemis/deleteDivert', {
```

Comment:
If the intent is not to delete a divert as per reply to previous comment, then this should not be added

Reply:
Well, it shows page without such button. But I agree it would be better to hide this button in "toolbar". I will take a look at this.